thermopy/nasa9polynomials.py

```diff
@@ -909,3 +909,4 @@ def __repr__(self):
         for (reag, coef) in zip(self._products, self._pcoefs):
             r = r + '+' + str(coef) + ' ' + reag.inp_name + ' '
         return """<reaction> {0}""".format(r)
+    print('test')
```

Comment:
Thanks! One last change request. Can you please remove this print statement too?